sopel/plugins/handlers.py

```diff
@@ -622,10 +624,14 @@ def get_version(self) -> Optional[str]:
         """
         version: Optional[str] = super().get_version()
 
+        # note: we need to check for attribute because of older Python version
```

Comment:
Should say which versions, so future-us can easily know when it's safe to refactor into something simpler.